src/util/requests/impl/WsConnectionImpl.hpp

```diff
@@ -134,7 +134,7 @@ class WsConnectionImpl : public WsConnection {
         // The timer below can be called with no error code even if the operation is completed before the timeout, so we
         // need an additional flag here
         timer.async_wait([&cancellationSignal, isCompleted](boost::system::error_code errorCode) {
-            if (!errorCode and not *isCompleted)
+            if (!errorCode and not*isCompleted)
```

Comment:
Should we apply the commit that fixes this ugly syntax?

Reply:
I think it doesn't matter for the release

Reply:
Fine 👍